docs/excel/excel-add-ins-ranges-clear-delete.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Clear or delete ranges using the Excel JavaScript API
+description: 'Learn how to clear or delete ranges using the Excel JavaScript API.'
+ms.date: 04/02/2021
+localization_priority: Normal
+---
+
+# Clear or delete ranges using the Excel JavaScript API
+
+This article provides examples that show how to clear and delete ranges with the Excel JavaScript API. For the complete list of properties and methods that the `Range` object supports, see the [Excel.Range class](/javascript/api/excel/excel.range).
```

Comment:
There's a cadence to all these new article intros that's like "This article provides code samples that show..." (or something similar). Consider reframing them to give the article itself the authority, like "This article shows you how to...".